### PR TITLE
Added styling for files missing

### PR DESCRIPTION
**Pull request by flamerohr**
Similar to "File failed to upload", it'll be a red background with white text over the "image" area.

Checkbox is selectable, so multiple missing files can be batch deleted if needed.

### CODE REVIEW

**Comment by clarkepaul** [comment]
CSS wise looks good

File: client/lang/en.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "AssetAdmin.EDIT": "Edit",
     "AssetAdmin.FILENAME": "Filename",
     "AssetAdmin.FILES": "Files",
+    "AssetAdmin.FILE_MISSING": "File cannot be found",
```
Comment (tractorcow) [inline client/lang/en.js]: This needs to be added to the source file, as this will be overwritten on update.

File: client/lang/en.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
...
     "AssetAdmin.FILTER_DATE_ASC": "oldest",
     "AssetAdmin.FILTER_DATE_DESC": "newest",
```
Comment (tractorcow) [inline client/lang/en.js]: Restarting review, didn't mean to approve this review. :D

**Comment by tractorcow** [comment]
I think this PR is blocked by https://github.com/silverstripe/silverstripe-asset-admin/issues/261.

@flamerohr can you please look at this tomorrow?

**Comment by tractorcow** [comment]
Related but not blocking issue at https://github.com/silverstripe/silverstripe-framework/pull/6056

File: client/lang/en.js (hunk 1 above)
Comment (chillu) [inline client/lang/en.js]: It's been added into `client/lang/src/en.js` now

**Pull request merged**